src/libs/actions/PaymentMethods.js

```diff
@@ -35,6 +35,34 @@ function getPaymentMethods() {
         });
 }
 
+/**
+ * Call the API to transfer wallet balance.
+ * @param {Object} paymentMethod
+ * @param {String} paymentMethod.id
+ * @param {'bankAccount'|'debitCard'} paymentMethod.type
+ * @returns {Promise}
+ */
+function transferWalletBalance(paymentMethod) {
+    const parameters = {
+        [paymentMethod.type === CONST.PAYMENT_METHODS.BANK_ACCOUNT ? 'bankAccountID' : 'fundID']: paymentMethod.id,
+    };
+    Onyx.merge(ONYXKEYS.WALLET_TRANSFER, {loading: true});
+
+    return API.TransferWalletBalance(parameters)
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                throw new Error(response.message);
```

Comment:
Just throw? Should we handle these errors and give the user some feedback?

Reply:
Error is handled in the Catch block.

Reply:
Ah I see. Would be good to standardize on this pattern. I see it done other ways most times.

Reply:
I like this pattern for the following reasons.
1. delegate the error handling to catch (which should be responsible for it).
2. reduce duplicate logic.

I am pretty sure it would be my idea in other places as well. :rofl: 

Should I change it to like the other parts of app?

Reply:
We should always try to do things however they are done elsewhere - but this feels like something we haven't established a standard for yet. I'll create an issue to look into this. 

The reasons you give make sense to me, but have a counterpoint that `catch()` for an API almost always indicates a networking issue in our code - so it's easier for me to think about what has happened to the user when handling is separated vs. combined like this.

Let's figure it out in another issue :)